metrics/Inactive_Contributers.md

```diff
@@ -0,0 +1,33 @@
+# Inactive_Contributers
+
+Question: How many Contributers have gone inactive over a specific period of time.
+
+## Description
+A metric that shows how many contributers have stopped committing over a specific period of time. The period of time required for a contributer to be counted as inactive can be decided by a variable and this metric will display the number of contributers that have been labeled as inactive over a given time frame.
+
+## Objectives
+The objective is to determine how many people are losing intrest in the project. This can be useful in determining intrest in the project. User intrest and user retention is an important metric for determing the health of a project and watching how many users are going inactive can help in finding those values. It is important to know if many people are leaving the project and have stopped commiting.
```

Comment:
Keep the first sentence, and delete the rest.

Reply:
Change the first sentence to "The objective is to determine how many people have stopped contributing actively. This could be useful for community managers to determine if key members are losing interest, or are taking a break.